specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -5660,6 +5720,10 @@
           "items": {
             "$ref": "#/definitions/InvoiceSection"
           }
+        },
+        "budgetInformation": {
+          "description": "The budget information associated with the billing profile.",
```

Comment:
It's associated with/scoped to the Azure plan, not the billing profile.

Reply:
As per discussion with folks, it was decided to keep this as scoped to BillingProfile and not plan.